test/box-luatest/walext_test.lua

```diff
@@ -0,0 +1,24 @@
+local misc = require('test.luatest_helpers.misc')
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.test_walext_unavailable = function()
+    misc.skip_if_enterprise()
+    g.server = server:new({alias = 'master'})
+    g.server:start()
+    g.server:exec(function()
+        local t = require('luatest')
+        t.assert_error_msg_equals(
+            "Community edition does not support WAL extensions",
+            box.cfg, {wal_ext = {}})
+    end)
+    g.server:exec(function()
+        local t = require('luatest')
+        local s = box.schema.create_space('t')
+        t.assert_error_msg_equals(
+            "Community edition does not support WAL extensions", s.wal_ext, s)
+        s:drop()
+    end)
+    g.server:drop()
```

Comment:
This should be done in an "after" callback so that the test case leaves no traces in case of failure. Also, please rename the file to `wal_ext_test` for consistency with the `wal_ext` configuration option.

Reply:
Fixed